Explorer/Assets/DCL/Multiplayer/Movement/Components/InterpolationComponent.cs

```diff
@@ -21,6 +24,13 @@ public void Restart(NetworkMovementMessage from, NetworkMovementMessage to, Inte
             Time = 0f;
             TotalDuration = End.timestamp - Start.timestamp;
 
+            int movementBlendId = AnimationMovementBlendLogic.GetMovementBlendId(End.velocity.sqrMagnitude, End.movementKind);
```

Comment:
It's minor but can you check if you call `magnitude` and `sqrmagnitude` multiple times for the same structure? Maybe it's worth caching them once upon creation to avoid unnecessary calculations?

E.g. in `RemotePlayerMovementComponent`, `Extrapolation`, etc...